.github/workflows/build-rust-client.yml

```
@@ -58,16 +58,16 @@ jobs:
                   folder: ./clients/rust
                   key: rust-client-test
 
-            - name: Run cargo clippy
+            - name: Run cargo clippy (default feature set)
               uses: actions-rs/cargo@v1
```

actionlint (1.7.12)

[error] the runner of "actions-rs/cargo@v1" action is too old to run on GitHub Actions. update the action's version to fix this issue

(action)